packages/astroport_periphery/src/lockdrop.rs

```diff
@@ -308,9 +310,9 @@ pub struct LockUpInfoResponse {
     /// ASTRO tokens receivable as generator rewards that user can claim
     pub claimable_generator_astro_debt: Uint128,
     /// Generator Proxy tokens lockup received as generator rewards
-    pub generator_proxy_debt: Uint128,
+    pub generator_proxy_debt: Vec<(AssetInfo, Uint128)>,
```

Comment:
I think we can use RestrictedVector here

Reply:
Changed, thanks